BoardUserInterface.API/Controllers/V1/TemplateController.cs

```diff
@@ -10,27 +11,29 @@ namespace BoardUserInterface.API.Controllers.V1
     public class TemplateController : ControllerBase
     {
         private readonly ITemplateService _templateService;
+        private readonly ILogger<FileService> _logger;
 
-        public TemplateController(ITemplateService templateService)
+        public TemplateController(ITemplateService templateService, ILogger<FileService> logger)
```

Comment:
logger is being used in the controller? if not remove please

Reply:
removed